### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by valeriia-vasylyshyna**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -3,8 +3,43 @@
...
+    expect(getHumanAge(24, 24))
+      .toEqual([2, 2]);
+  });
+
+  it('should return 3 if cat`s age = 28  and dog`s age 29', () => {
+    expect(getHumanAge(28, 29))
+      .toEqual([3, 3]);
+  });
+
+  it('should return 3 if cat`s age = 32 and dog`s age = 34', () => {
```
Comment (murat-yesimbietov) [inline src/getHumanAge.test.js:36]: Do you mean 4?

File: src/getHumanAge.test.js [hunk 2]
```diff
@@ -3,8 +3,43 @@
...
+    expect(getHumanAge(14, 14))
+      .toEqual([0, 0]);
+  });
+
+  it('should return 1 if ages of a cat and a dog = 15', () => {
+    expect(getHumanAge(15, 15))
+      .toEqual([1, 1]);
+  });
+
+  it('should return 2 if ages of a cat and a dog = 24', () => {
```
Comment (murat-yesimbietov) [inline src/getHumanAge.test.js:26]: Add getHumanAge(23, 23) checking (boundary values technique)

File: src/getHumanAge.test.js [hunk 3]
```diff
@@ -3,8 +3,43 @@
...
+    expect(getHumanAge(24, 24))
+      .toEqual([2, 2]);
+  });
+
+  it('should return 3 if cat`s age = 28  and dog`s age 29', () => {
```
Comment (murat-yesimbietov) [inline src/getHumanAge.test.js:31]: Add getHumanAge(27, 28) checking

**Review by murat-yesimbietov — CHANGES_REQUESTED** [review]

File: src/getHumanAge.test.js (hunk 2 above)
Comment (valeriia-vasylyshyna) [reply src/getHumanAge.test.js]: Done

File: src/getHumanAge.test.js (hunk 3 above)
Comment (valeriia-vasylyshyna) [reply src/getHumanAge.test.js:41]: Done

File: src/getHumanAge.test.js (hunk 1 above)
Comment (valeriia-vasylyshyna) [reply src/getHumanAge.test.js]: Yes, I`ve changed  it

**Review by murat-yesimbietov — APPROVED** [review]
